Makefile

```
@@ -0,0 +1,43 @@
+            "      <port>$$port</port>" \
+            '    </proxy>' \
+            '  </proxies>' \
+            '</settings>' > $$settings; \
+        MAVEN_OPTS="$(MAVEN_JAVA_OPTS) -Dhttp.proxyHost=$$host -Dhttp.proxyPort=$$port -Dhttps.proxyHost=$$host -Dhttps.proxyPort=$$port $$MAVEN_OPTS" mvn --settings $$settings $(MAVEN_ARGS) $(1); \
+        rm $$settings; \
+    else \
+        MAVEN_OPTS="$(MAVEN_JAVA_OPTS) $$MAVEN_OPTS" mvn $(MAVEN_ARGS) $(1); \
+    fi
+endef
+
+.PHONY: test verify precommit check-docs
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)